src/PubSub/PubSubClient.php

```diff
@@ -49,7 +49,7 @@
  * ```
  *
  * Please take care in installing the same version of these libraries that are
- * outlined in the project's composer.json suggest keyword.
+ * outlined in the project's composer.json require-dev keyword.
```

Comment:
We should still be referring to the suggests rather than require-dev. Suggests includes the descriptions of what they can be added for.